app/controllers/releases_controller.rb

```diff
@@ -34,10 +34,30 @@ def create
   def update
     @release = Release.find(params[:id])
 
-    if @release.update_attributes(params[:release])
+    if @release.update_attributes(release_params)
       redirect_to @release, flash: { notice: "Successfully updated the release" }
     else
       redirect_to @release, flash: { alert: "There are some problems with the release" }
     end
   end
+
+  private
+    def release_params
+      params.require(:release).permit(
+        :additional_support_notes,
+        :deploy_at,
+        :id,
+        :notes,
+        :product_team_members,
+        :summary,
+        tasks_attributes: [
```

Comment:
Oh, it's an array not a hash! That's where I was going wrong! And it has way more things in it than I thought... :-P